manifests/4.9/aws-efs-csi-driver-operator.v4.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,332 @@
+    name: Red Hat
+  labels:
+    alm-owner-metering: aws-efs-csi-driver-operator
+    alm-status-descriptors: aws-efs-csi-driver-operator.v4.9.0
+  selector:
+    matchLabels:
+      alm-owner-metering: aws-efs-csi-driver-operator
+  installModes:
+  - type: OwnNamespace
+    supported: false
+  - type: SingleNamespace
+    supported: false
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+        - rules:
+          - apiGroups:
+            - ''
+            resources:
+            - pods
+            - services
+            - endpoints
+            - events
+            - configmaps
+            - secrets
+            verbs:
+            - '*'
+          - apiGroups:
+            - apps
+            resources:
+            - deployments
+            - daemonsets
+            - replicasets
+            verbs:
+            - '*'
+          serviceAccountName: aws-efs-csi-driver-operator
+      clusterPermissions:
+        - rules:
+          - apiGroups:
+            - security.openshift.io
+            resourceNames:
+            - privileged
+            resources:
+            - securitycontextconstraints
+            verbs:
+            - use
+          - apiGroups:
+            - operator.openshift.io
+            resources:
+            - clustercsidrivers
+            verbs:
+            - get
+            - list
+            - watch
+            # The Config Observer controller updates the CR's spec
+            - update
+            - patch
+          - apiGroups:
+            - operator.openshift.io
+            resources:
+            - clustercsidrivers/status
+            verbs:
+            - get
+            - list
+            - watch
+            - update
+            - patch
+          - apiGroups:
+            - ''
+            resourceNames:
+            - extension-apiserver-authentication
+            - aws-ebs-csi-driver-operator-lock
+            resources:
+            - configmaps
+            verbs:
+            - '*'
+          - apiGroups:
+            - rbac.authorization.k8s.io
+            resources:
+            - clusterroles
+            - clusterrolebindings
+            - roles
+            - rolebindings
+            verbs:
+            - watch
+            - list
+            - get
+            - create
+            - delete
+            - patch
+            - update
+          - apiGroups:
+            - ''
+            resources:
+            - serviceaccounts
+            verbs:
+            - get
+            - list
+            - watch
+            - create
+            - update
+            - patch
+            - delete
+          - apiGroups:
+            - coordination.k8s.io
+            resources:
+            - leases
+            verbs:
+            - '*'
+          - apiGroups:
+            - ''
+            resources:
+            - nodes
+            verbs:
+            - '*'
```

Comment:
does the operator needs all permissions for nodes?

Reply:
Removed